site/content/en/docs/Guides/Best Practices/gke.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "GKE Best Practices"
+linkTitle: "GKE"
```

Comment:
Should we make the linkTitle "Google Cloud" to be consistent with how we do the cloud specific naming in the cluster creation section?

Reply:
I never noticed that dissonance but I changed it over and expanded GKE the first use.